app/config/FeatureSwitches.scala

```diff
@@ -40,6 +42,14 @@ case class FeatureSwitches(featureSwitchConfig: Configuration) {
   val isTaxYearNotEndedRuleEnabled: Boolean = isEnabled("taxYearNotEndedRule.enabled")
   val isTaxYearSpecificApiEnabled: Boolean  = isEnabled("tys-api.enabled")
 
+  def isTemporalValidationEnabled(implicit request: Request[_]): Boolean = {
+    if (isEnabled("allowTemporalValidationSuspension.enabled")) {
```

Comment:
good practice to declare this config in the main application.conf file as well

Reply:
Oh totally missed that, good spot